### ISSUE TITLE
running after installing crashes

### ISSUE THREAD

**Issue by bnvk** [adafruit/nprone-cli#1]
I'm running node version 5.11.1 on Debian, after installing your package successfully and running it, I get the following error:

```
/usr/lib/node_modules/npr-one/node_modules/npr-api/lib/helpers.js:51
  return (...args) => {
          ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/lib/node_modules/npr-one/node_modules/npr-api/lib/one.js:4:17)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
```

**Comment by toddtreece** [comment]
sorry, there should have been a note in the readme about this. it requires node 6.x due to the use of es6 features like the spread operator `...args`.

fixed with 96d1e7fbf98c0c217e67da87284a1730c56551e0

**Comment by toddtreece** [comment]
@bnvk i forgot to mention, the easiest way to get 6.x installed for you is probably something like

```sh
$ sudo npm install -g n && sudo n stable
```
